src/stylesheets/components/_mobile-navigation.scss

```diff
@@ -0,0 +1,129 @@
+.app-c-mobile-nav {
+  display: none;
+
+  &--active {
+    display: block;
+
+    @include mq($from: desktop) {
+      display: none;
+    }
+  }
+
+  &.app-c-mobile-nav--hide-on-pageload {
```

Comment:
I don't understand what this is doing when `app-c-mobile-nav` is `display: none` already?

Reply:
This is intended to hide the mobile nav until `no-js` html class gets removed with Modernizr. However I've just realised that it's working as intended 👎  I'll raise a card to revisit this.